Dart/src/com/jetbrains/lang/dart/analyzer/DartAnalysisServerService.java

```diff
@@ -140,6 +143,14 @@
   public long maxMillisToWaitForServerResponse = 0L;
   @NotNull private final InteractiveErrorReporter myErrorReporter = new InteractiveErrorReporter();
 
+  @NotNull private final Object myDebugLogLock = new Object();
```

Comment:
I double-checked this and found that it works as expected. Removing the lock field is fine. I can't explain why it was complaining yesterday. I can change this when I get back to work after the holidays.